vulcan/lib/client/jsx/contexts/api.ts

```diff
@@ -188,14 +188,13 @@ export function useApi(
       });
   }, [vulcanPost, vulcanPath]);
 
-  const createWorkspace = useCallback((projectName: string, workflowId: number, workspaceName: string, branch: string, git_version: string): Promise<CreateWorkspaceResponse> => {
+  const createWorkspace = useCallback((projectName: string, workflowId: number, workspaceName: string, git_request: string): Promise<CreateWorkspaceResponse> => {
     return vulcanPost(
       vulcanPath(`/api/v2/${projectName}/workspace/create`),
       {
         workflow_id: workflowId,
-        branch: branch,
+        git_request: git_request,
         workspace_name: workspaceName,
-        git_version: git_version,
       });
```

Comment:
Updated api request is this bit!  I'd added `branch` erroneously in the big prev PR even though it was never used.... So compared to what you'd previously built towards, this PR just changes the 'git_version' key to 'git_request'.